src/Cms/Blueprint.php

```diff
@@ -345,6 +340,12 @@ public static function load(string $name): array
 		// inject the filename as name if no name is set
 		$props['name'] ??= $name;
 
+		// normalize the title
+		$title = $props['title'] ?? ucfirst($props['name']);
+
+		// translate the title
+		$props['title'] = I18n::translate($title) ?? $title;
```

Comment:
In other places we use `I18n::translate($title, $title)` to fix exactly that issue. Because then the `translate` method handles it by taking the first value from the array. As far as I remember we had that before here and you switched to `??`. What was the reason for that again?

Reply:
Haha yes that's the beginning of the whole journey. When we pass the same array as fallback, we ended up with the problem that i18n keys would not use the default's language value from the global translations strings as fallback.